discover/portal/articles/130-forms/02-managing-form-entries.markdown

```diff
@@ -0,0 +1,94 @@
+
+## Viewing Form Entries [](id=viewing-form-entries)
+
+When users fill out forms, they're generating data. You're going to want to see
+it at some point. 
+
+1.  From the Menu, navigate back to the *Content* &rarr; *Forms* section of The
+    Lunar Resort site.
+
+2.  Click the *Actions* (![Actions](../../images/icon-actions.png)) button and
+    select *View Entries*.
+
+    ![Figure 8: You can view the entries right in the Forms application. ](../../images/forms-view-entries.png)
+
+3.  What if you  have a lot of form fields and you can't see all of the data for
+    each entry in the search container? Just click the *Actions*
+    (![Actions](../../images/icon-actions.png)) button for the entry and select
+    *View*. You'll be able to see all the specifics for that form entry.
+
+    ![Figure 9: You can view a single entry right in the Forms application.](../../images/forms-view-entry.png)
+
+Viewing entries is great, but this is serious data we're talking about. You
+might need to get all the entries into a spreadsheet so you can work with them,
+for example.
+
+## Exporting Form Entries [](id=exporting-form-entries)
+
+So, you need to put your form entries in a spreadsheet to do spreadhseet things
+with them? No problem.
+
+1.  Navigate to the Forms application in The Lunar Resort site's Content section
+    again.
+
+2.  Click the *Actions* (![Actions](../../images/icon-actions.png)) button and
+    select *Export*. 
+
+3.  Choose a File Extension. You can get entries in CSV, XLS, or XML. For this
+    example, pick CSV.
+
+4.  Click *Okay*, and open the file or save it locally. Open it with your
+    favorite spreadsheet program and confirm that your form entries are entered
+    into a spreadsheet for you.
+
+![Figure 10: You can export entries as CSV, XLS, or XML.](../../images/forms-export-csv.png)
+
++$$$
+
+**Note:** The Forms application itself has an *Import/Export* window that you can
+access from the application's Configuration menu
+(![Configuration](../../images/icon-options.png)). This is how you import and
+export the application configuration and its data (forms and form entries). The
+file format for this type of import and export is a LAR file. For more
+information, see the article on
+[importing and exporting application content.](/discover/portal/-/knowledge_base/7-1/importing-exporting-pages-and-content).
+
+$$$
+
+Once you export a batch of form entries, it can make sense to delete them from
+the database.
+
+## Deleting Form Entries [](id=deleting-form-entries)
+
+What if you export a form's entries and now you want them to be removed from the
+Liferay database? It's easy to delete all of a form's entries at once.
+
+Navigate back to the Forms application In The Lunar Resort Content section.
+Click the *Actions* (![Actions](../../images/icon-actions.png)) button next
+to the Guest Survey form and select *View Entries* again. Select all of the
+entries by checking the box next to *All*. A trash can icon
```

Comment:
On master, the text next to the box isn't **All** anymore, it was changed to **Filter and Order**, please see it below:

![screen shot 2018-05-15 at 11 48 43 am](https://user-images.githubusercontent.com/6729287/40064581-34d2ed24-5836-11e8-88bf-0731776b9aae.png)